webapp/i18n/ru.json

```diff
@@ -415,7 +415,7 @@
   "admin.ldap.firstnameAttrDesc": "(Опционально) Этот атрибут для AD/LDAP сервера который будет использоваться для заполнения имени пользователя в Mattermost. Если он установлен, пользователи не смогут редактировать свои имена т.к. они будут синхронизироваться с LDAP сервером. Если атрибут не заполнен, пользователи смогут изменить их собственные имена в Настройках пользователя.",
   "admin.ldap.firstnameAttrEx": "Например: \"givenName\"",
   "admin.ldap.firstnameAttrTitle": "Атрибут имени",
-  "admin.ldap.idAttrDesc": "Атрибут LDAP, который будет использоваться в качестве уникального идентификатора. Укажите атрибут, значение которого не будет меняться, например имя пользователя или его uid. Если значение этого атрибута изменится, в Mattermost будет создана новая учетная запись, не имеющая к старой никакого отношения. Значение этого атрибута используется в поле \"Имя пользователя LDAP\" на странице входа в Mattermost. В большинстве случаев это тот же атрибут, что используется в качестве \"Атрибута имени пользователя\" выше. Если ваша команда обычно использует учетные записи в стиле домен\\пользователь для входа в другие сервисы с использованием LDAP, вы можете указать это же значение и здесь, для сохранения единообразия.",
+  "admin.ldap.idAttrDesc": "Атрибут LDAP, который будет использоваться в качестве уникального идентификатора. Укажите атрибут, значение которого не будет меняться, например имя пользователя или его uid. Если значение этого атрибута изменится, в Mattermost будет создана новая учетная запись, не имеющая к старой никакого отношения. Значение этого атрибута используется в поле “Имя пользователя LDAP“ на странице входа в Mattermost. В большинстве случаев это тот же атрибут, что используется в качестве \"Атрибута имени пользователя\" выше. Если ваша команда обычно использует учетные записи в стиле домен\\\\пользователь для входа в другие сервисы с использованием LDAP, вы можете указать это же значение и здесь, для сохранения единообразия.",
```

Comment:
Thanks @ArchRoller! 

Can you help submit the update in [the Translation Server](http://translate.mattermost.com/ru/mattermost/translate/#search=%D0%90%D1%82%D1%80%D0%B8%D0%B1%D1%83%D1%82%20LDAP%2C%20%D0%BA%D0%BE%D1%82%D0%BE%D1%80%D1%8B%D0%B9%20%D0%B1%D1%83%D0%B4%D0%B5%D1%82%20%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D1%8C%D1%81%D1%8F%20%D0%B2%20%D0%BA%D0%B0%D1%87%D0%B5%D1%81%D1%82%D0%B2%D0%B5&sfields=source,target) and it'll be included in the next pull request?

Otherwise the string will be overwritten later by what is currently on the Translation Server,